Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -1533,6 +1533,20 @@ The mini form task is used to test survey forms functionality (`ORKFormStep`).
     }
     
     {
+        /*
+         A text question with single-line text entry, text validation, and a URL keyboard.
+         */
+        ORKTextAnswerFormat *format = [ORKAnswerFormat textAnswerFormatWithValidationExpression:@"^(https?:\\/\\/)?([\\da-z\\.-]+)\\.([a-z\\.]{2,6})([\\/\\w \\.-]*)*\\/?$" validInputDescription:@"Enter a valid URL."];
+        format.multipleLines = NO;
+        format.keyboardType = UIKeyboardTypeURL;
+        format.autocapitalizationType = UITextAutocapitalizationTypeNone;
```

Comment:
You also might want to turn off spell checking and auto correction. Every time I typed in a URL, and hit NEXT, it would proceed, but when I came back to the step, it was auto corrected to something else. Sometimes even something else that is no longer valid.